### ISSUE TITLE
[symfony] - Build Error with SSL Harding

### ISSUE THREAD

**Issue by DocCyblade** [turnkeylinux/tracker#445]
Error when building with SSL Harding, not sure if it's related or not.

```
root@tkldev-l products/symfony# git branch -v
* ssl-hardening fe30554 rm cert from apache conf (now in mods/ssl.conf)
https://github.com/JedMeister/symfony.git
```

Testing Tracking here (https://github.com/turnkeylinux/common/pull/42#issuecomment-132832224)

#### Error Message below
```
+ chown -R www-data:www-data /var/www/symfony/app/cache
+ chown -R www-data:www-data /var/www/symfony/app/logs
+ HTACCESS=/var/www/symfony/web/.htaccess
+ cat
+ APPDEV=/var/www/symfony/web/app_dev.php
+ cat
+ TEMPLATE=/var/www/symfony/src/Acme/DemoBundle/Resources/views/Welcome/index.html.twig
+ sed -i s|<h1>\(.*\)</h1>|<h1>Welcome to TurnKey Symfony</h1>| /var/www/symfony/src/Acme/DemoBundle/Resources/views/Welcome/index.html.twig
sed: can't read /var/www/symfony/src/Acme/DemoBundle/Resources/views/Welcome/index.html.twig: No such file or directory
/usr/share/fab/product.mk:476: recipe for target 'build/stamps/root.patched' failed
make: *** [build/stamps/root.patched] Error 2

```

**Comment by DocCyblade** [comment]
Wrote cakePHP by mistake, this is for symfony

**Comment by DocCyblade** [comment]
I think this is due to a new version, and it may have changed some files. Not sure but I know sf has changed in few years.

**Comment by DocCyblade** [comment]
The new LTS is 2.7, 2.3 is still supported but we should move to 2.7. I have added an 

They also changed the installer to not include the demo bundle by default. 

Easy fix, would be to make sure your downloading the demo bundle, I would however like to see 2.7 for v14 release or v14.1. I have added a feature request for LTS 2.7

**Comment by DocCyblade** [comment]
submitted a feature request (https://github.com/turnkeylinux/tracker/issues/447) for LTS 2.7

As I did for lime survey I'll fork @JedMeister symfony repo update the ssl branch to get it to build and test, submit a PR request that would close this issue. Should be done today

**Comment by JedMeister** [comment]
Awesome :+1:

**Comment by DocCyblade** [comment]
Confirmed the demo app has changed as did 2.3 to 2.7 LTS.  I was able to get it to build by removing edits that are made to the welcome page. The whole demo has changed, also the demo seems to be only LTS 2.7. This could be a good thing. I was able to get it to build to test the SSL but it does not "TKL Brand" the first page. I did a full ISO build and it seems to work but getting some HTTP 401 errors. I'll finish my SSL testing and will take a closer look.
